.registry/resources/minimalaws.ui-schema.yaml

```diff
@@ -13,35 +13,26 @@ configSections:
     validation:
     - minLength: 2
     - maxLength: 62
-  - name: credentialSecretName
-    controlType: singleInput
-    type: string
-    path: .spec.credentialsSecretRef.name
-    title: Credentials Secret Name
-    pattern: ^([A-Za-z0-9](?:(?:[-A-Za-z0-9]){0,61}[A-Za-z0-9])?){2,62}$
-    description: The name of the secret that you stored your AWS credentials.
-    validation:
-    - minLength: 2
-    - maxLength: 62
-  - name: credentialSecretNamespace
-    controlType: singleInput
-    type: string
-    path: .spec.credentialsSecretRef.namespace
-    title: Credentials Secret Namespace
-    pattern: ^([A-Za-z0-9](?:(?:[-A-Za-z0-9]){0,61}[A-Za-z0-9])?){2,62}$
-    description: The namespace of the secret that you stored your AWS credentials.
-    validation:
-    - minLength: 2
-    - maxLength: 62
-  - name: credentialSecretKey
-    controlType: singleInput
-    type: string
-    path: .spec.credentialsSecretRef.key
-    title: Credentials Secret Data Key
-    pattern: ^([A-Za-z0-9](?:(?:[-A-Za-z0-9]){0,61}[A-Za-z0-9])?){2,62}$
-    description: The key that has the credential data in the secret.
-    validation:
-    - minLength: 2
-    - maxLength: 62
-printerColumns:
-  Ready: .status.conditionedStatus.conditions[?(@.type=='Synced')]
\ No newline at end of file
+    - required: true
+- title: Generate Provider Secret
+  description: You can use the following command to generate an AWS Provider secret that contains your default profile credentials.
+  items:
+    - name: additionalInfoSnippet
+      controlType: codeSnippet
+      codeText: aws_profile=default && echo -e "[default]\naws_access_key_id = $(aws configure get aws_access_key_id --profile $aws_profile)\naws_secret_access_key = $(aws configure get aws_secret_access_key --profile $aws_profile)"
```

Comment:
are we sure this command will work for anyone running it?

Reply:
It assumes that the profile you'd like to use is named `default`, which is what you get when you do `aws init`. I'll add some info about `aws_profile` env var to tell user what it's for and they can configure it.